source/scenes/home/SendAsset/Confirm/Confirm.container.tsx

```diff
@@ -90,8 +90,9 @@ const ConfirmContainer = () => {
       to
     } = queryString.parse(location.search);
 
+
     activeAsset = useSelector(
-      (state: RootState) => find(state.assets, { address: to })
+      (state: RootState) => find(state.assets, { address: Array.isArray(to) ? to[0] : to })
```

Comment:
Doesn't this introduce the possibility of an asset that we don't have in state being passed as the `to` variable? It would then fail below on line 105.

Reply:
Ummm (I hope I understood correctly), that was more of a type guard around the `queryString.parse` method, which returns a `string | string[]` type. It should not affect anything below as we're not really passing anything we don't know about first-hand.

Reply:
This doesn't ensure that an asset in state is the `activeAsset` which is what we assume below. The previous code included that check.

Reply:
Never mind. I was confused. We're still finding the activeAsset in state.